src/components/TradeWidget/index.tsx

```diff
@@ -370,13 +393,17 @@ const TradeWidget: React.FC = () => {
   const [sellToken, setSellToken] = useState(
     () =>
       trade.sellToken ||
-      getToken('symbol', sellTokenSymbol, tokens) ||
+      (sellTokenSymbol && isAddress(sellTokenSymbol?.toLowerCase())
```

Comment:
Can we refactor this a bit, also it grow the oneline statement a bit. I think it's the moment to add some more lines/statements/intermediate-vars or whatever it takes to read it simpler